src/AirshipBindings.iOS.Accengage/urbanairship.ios.accengage.targets

```diff
@@ -0,0 +1,14 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Target Name="AirshipAccengageBeforeCompile" AfterTargets="BeforeCompile">
+        <ItemGroup>
+            <BindingResources Include="$(MSBuildThisFileDirectory)../content/AirshipBindings.iOS.Accengage.resources/**/*.*" />
+        </ItemGroup>
+        <Copy SourceFiles="@(BindingResources)" DestinationFolder="$(TargetDir)/AirshipBindings.iOS.Accengage.resources/%(RecursiveDir)" ContinueOnError="false" />
+        <ItemGroup>
+            <NativeReference Include="$(TargetDir)/AirshipBindings.iOS.Accengage.resources\AirshipAccengage.xcframework">
```

Comment:
This makes VS link against the framework directly (instead of implicitly through the DLL) if someone imports the nupkg into their project